src/components/page-components/ExploreProjectPage/ExploreProjectPage.tsx

```diff
@@ -0,0 +1,57 @@
+import { sendToBackground } from "@plasmohq/messaging"
+
+import "./ExploreProjectPage.css"
+
+import { useEffect, useState } from "react"
+
+import BackButton from "~components/BackButton/BackButton"
+import CuratorDetails from "~components/CuratorDetails/CuratorDetails"
+import Footer from "~components/Footer/Footer"
+import ProjectDetails from "~components/ProjectDetails/ProjectDetails"
+import type { ProjectData } from "~types/projectTypes"
+
+const ExploreProjectPage = () => {
+  const [project, setProject] = useState<ProjectData>()
+
+  useEffect(() => {
+    const fetchProject = async () => {
+      const response = await sendToBackground({
+        name: "fetchProjectDetailsById",
+        body: {
+          id: 1
+        }
+      })
+      console.log({ response })
+
+      setProject(response.data as ProjectData)
+    }
+    fetchProject()
+  }, [])
+
+  return (
+    <>
+      {project && (
+        <main className="explore-project page">
+          <BackButton className="explore-project__back-button" />
+          <img
+            src={
+              process.env.PLASMO_PUBLIC_API_URL +
+              project.cover_image.formats.small.url
+            }
+            alt={
+              project.cover_image.alternativeText ||
+              "Project image thumbnail"
```

Comment:
I'm not a stickler for this but maybe we could use the project title or something. I recall being told once that generic descriptors are not super screen-reader-friendly and maybe it would be better for the user to hear the title of the project they are hovering rather than "image thumbnail"